crates/fluvio-cluster/src/cli/start/local.rs

```diff
@@ -28,7 +28,7 @@ pub async fn process_local(
             builder.local_chart(chart_location);
         }
         None if opt.develop => {
-            builder.local_chart("./k8-util/helm");
+            builder.local_chart("./k8-util/helm/fluvio-sys");
```

Comment:
Hmm.  This is no longer correct. Let's just remove this option

Reply:
Do you mean not having a "default" value for local_chart when running --local in --develop?

Reply:
Yes. leave it optional